skyve-war/src/main/java/modules/admin/router.xml

```diff
@@ -3,7 +3,7 @@
 			xsi:schemaLocation="http://www.skyve.org/xml/router ../../schemas/router.xsd"
 			xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
 	<uxui name="external">
-		<route outcome="/external/public.xhtml">
+		<route outcome="/external/selfRegister.xhtml">
```

Comment:
I can't get this to work, it never routes for me from the login page. It just flashes and goes nowhere.